cypress/e2e/features/Ui/designServices.feature

```diff
@@ -19,4 +21,4 @@ Feature: Design Services
         And User fills all necassary fields
         Then User clicks summit button
         Then User verifies success message
-            """
```

Comment:
why did you delete line 22? it will give error in feature file. I think you tried to resolve conflicts and something got messy :) Could you please check this file in main branch and push again as in the main branch